app/uk/gov/hmrc/eoricommoncomponent/frontend/config/AppConfig.scala

```diff
@@ -60,10 +60,12 @@ class AppConfig @Inject() (
   lazy val externalGetEORILink = config.get[String]("external-url.get-cds-eori")
 
   lazy val blockedRoutesRegex: Seq[Regex] = {
-    config.getOptional[String]("routes-to-block") match {
+    val eoriSubscribe = "eori/subscribe".r
```

Comment:
Is this necessary?

Reply:
No.  Left over from previous soln.  Good spot

Reply:
Fixed!